spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientSecretsPropertySourceLocator.java

```diff
@@ -90,8 +90,8 @@ else if (kubernetesClientProperties != null) {
 					kubernetesNamespaceProvider);
 		}
 
-		return new KubernetesClientSecretsPropertySource(coreV1Api, secretName, namespace, environment,
-				normalizedSource.getLabels(), this.properties.isFailFast());
```

Comment:
environment is not needed - so I removed it, this in turn triggered some changes in some other places; mainly in test classes